releasenotes/notes/adds_numpyeigensolver_transition_amplitudes-478c18d4e517b564.yaml

```diff
@@ -0,0 +1,8 @@
+features:
+  - |
+    Adds the public method :meth:`.NumpyEigensolver.compute_transition_amplitudes` to
+    the :class:`.NumpyEigensolver`. This allows the user to specify auxiliary operators
+    and pairs of eigenstates (k,l) to evaluate the associated transition amplitudes after 
+    :meth:`.NumpyEigensolver.compute_eigenvalues` was called. See the docstring for an example
```

Comment:
> after :meth:`.NumpyEigensolver.compute_eigenvalues` was called

This is quite an important detail which is not documented in the method's docstring. Maybe you should either guard against `self._ret` being empty or add `operator` as an argument and call `compute_eigenvalues` yourself, when it hasn't been called yet.

Reply:
Good point